# supplementary-GlobalEuphorbiaStudy

This repository contains supplementary materials relating to the manuscript 
"The chemoevolution of diterpenoids in <i>Euphorbia</i> across the world" by M. Ernst ([@madeleineernst](https://github.com/madeleineernst)), L.-F. Nothias ([@lfnothias](https://github.com/lfnothias)), J. J. J. van der Hooft, R. R. Silva ([@rsilvabioinfo](https://github.com/rsilvabioinfo)), C. H. Saslis-Lagoudakis, 
O.  M. Grace, K. Martinez-Swatson, G. Hassemer, L. A. Funez, H. T. Simonsen, M. H. Medema, 
D. Staerk, N. Nilsson, P. Lovato, P. C. Dorrestein & N. Rønsted.

# Citation

 Please cite this supplementary material through: https://doi.org/10.1101/323014.

# Files

### Cytoscape_SummaryTable.csv

The file called **Cytoscape_SummaryTable.csv** is a summary table of the mass spectral molecular network data described in the the manuscript and the supplementary material.

### Euphorbia_Network.cys

The file called **Euphorbia_Network.cys** is a file, which can be opened in Cytoscape version 3.4.0 ([Shannon et al., 2003](https://genome.cshlp.org/content/13/11/2498.full)) and visualizes the mass spectral molecular network and associated data as described in the manuscript and the supplementary material.


# Folders

## 3DMolecularCartography

The folder called **3DMolecularCartography** contains R scripts and data files used for analyses associated with the 3D mass spectral molecular cartography. For more details see `3DMolecularCartography/README`.


## ClassyFire

The folder called **ClassyFire** contains R scripts and data files used for performing automated chemical classification of the <i>in silico</i> annotated structures using ClassyFire ([Djoumbou Feunang et al., 2016](https://jcheminf.springeropen.com/articles/10.1186/s13321-016-0174-y)).

## Figure 1

The folder called **Figure 1** contains R scripts and data files used for creating Figure 1 of the manuscript.

## Figure 4

The folder called **Figure 4** and subfolders **Figure4A** and **Figures4BtoD** contain R scripts and data files used for creating Figures 4A to D of the manuscript.

## Mass2Motifs_2_MolecularNetwork

The folder called **Mass2Motifs_2_MolecularNetwork** contains R scripts and data files used for mapping Mass2Motifs ([Van der Hooft et al., 2016](http://www.pnas.org/content/113/48/13738.full); [Wandy et al., 2018](https://academic.oup.com/bioinformatics/article/34/2/317/4158166)) on the mass spectral molecular networks ([Wang et al., 2016](https://www.nature.com/articles/nbt.3597); [Watrous et al., 2012](http://www.pnas.org/content/109/26/E1743)). 

## Optimus

The folder called **Optimus** contains data files used and retrieved from LC-MS/MS data preprocessing through Optimus ([https://github.com/MolecularCartography/Optimus](https://github.com/MolecularCartography/Optimus)).

## PGLS

The folder called **PGLS** contains R scripts and data files used for performing the phylogenetic generalized least squares regression analysis (PGLS).

## Sirius_CSIFingerID

The folder called **Sirius_CSIFingerID** contains data files used and retrieved from <i>in silico</i> structure annotation through SIRIUS+CSI:FingerID ([Dührkop and Böcker, 2016](https://link.springer.com/chapter/10.1007%2F978-3-319-16706-0_10); [Dührkop et al., 2015](http://www.pnas.org/content/112/41/12580)). For more details see `Sirius_CSIFingerID/README`.
